### ISSUE TITLE
Allow Billing Address form in `order-pay` page

### ISSUE THREAD

**Issue by Kryptonit3** [woocommerce/woocommerce#21425]
**Is your feature request related to a problem? Please describe.**

Some businesses have no need for a front facing store / shop / product listings. They provide a service to customers and would like to use this product for invoicing and receiving payments (like I would). For the most part this product works as advertised. Here is the work flow:

Order is manually created in admin panel.
Invoice is emailed to client.
Client receives email and clicks pay link to pay invoice.
They are directed to the `order-pay` page.
Some payment gateways require address / zip as an extra layer of security.
Generating order invoices in this manor does NOT allow the customer to edit billing address details on this page.
Customer can edit their Billing Address on `My Account -> Addresses` page but doing so does not update existing invoice orders. ([Here is a work around I found and modified](https://wp.zacgordon.com/2017/06/07/update-woocommerce-order-address-dynamically-when-customer-updates-their-address/comment-page-1/#comment-5800))

**Describe the solution you'd like**
Just like the Cart / Checkout page, would like the option to enable Billing Address editing on the `order-pay` page.

**Describe alternatives you've considered**
Posting a notice to customers in the stripe description (like below) to edit their billing address before paying, and then [using this hack](https://wp.zacgordon.com/2017/06/07/update-woocommerce-order-address-dynamically-when-customer-updates-their-address/comment-page-1/#comment-5800) to update the `pending payment` order(s).

```
Pay with your credit card via Stripe. Verify your <a href="https://my-site/my-account/edit-address/">Billing Address</a> in <a href="https://my-site/my-account/">My Account</a> to avoid issues.
```

**Comment by DavidAnderson684** [comment]
Allowing customers to unilaterally re-issue an altered invoice to themselves can cause other administrative or legal/audit problems. This sounds more like extension territory rather than something that everybody would want to be default behaviour?

**Comment by Kryptonit3** [comment]
The customers don’t issue or reissue anything. The business builds the invoice and sends it to the customer. At the time of the build we don’t know what their billing address will be for payment method. When customer gets invoice email they are sent to the order-pay endpoint. They are unable to update or set a billing address. Changing it in customer dashboard doesn’t update outstanding invoices. And payment requests don’t work at this endpoint url.

**Comment by GAskygirl** [comment]
I came here to ask about this... it would help our scenario as well.

**Comment by KratosGemini** [comment]
I'd argue this is not as niche as it first seems. (We already ran into this and we only started using WooCommerce this year.) To me it seems like a UX error to have a customer enter in and/or choose their payment information but not allow them to add or change their billing address (not talking about shipping address). At best, admins can load their currently saved billing address, but this presents problems:

1. What if the customer needs to add a new payment method that has a different billing address?
2. What if the customer's billing address for a saved payment changed?
3. What if this is a new customer that has not placed and order yet and thus does not have a billing address entered at all?

We are perhaps partially saved by the fact that people don't move all the time, but it does happen. And when it happens or when a website needs to invoice a new customer, this becomes an issue.

Maybe I'm missing something....because this seems like a bigger issue than implied here.

**Comment by ThunderBunnyCo** [comment]
I agree with Kratos. We have people that call and want us to put the items together they need for their project. The order-pay screen would be perfect for this but it doesn't allow them to add their billing address or customer information. I don't know what the point of paying for an order with no billing or shipping address would be. Seems like an oversite on the developers.

**Comment by GAskygirl** [comment]
Our use case is very few customers have accounts, most check out as guests
and sometimes they want to order and pay over the phone. We like using
address verification for credit cards, so if we could create an order and
send them the payment page with billing and address info, it would be
perfect.

----------------------------------------------------
Blue Skies!
Christy
